tests/test_wave_01.py

```diff
@@ -142,13 +138,9 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert updated_data["watched"][1]["title"] == movie_to_watch["title"]
```

Comment:
Nice assert! Consider making it a bit more robust by checking the whole movie instead of just the title.